packages/typespec-python/tasks.py

```diff
@@ -110,6 +110,7 @@
         "package-name": "client-structure-twooperationgroup",
     },
     "mgmt/sphere": [
+        {"package-name": "azure-mgmt-spherejson", "models-mode": "none"},
```

Comment:
we really shouldn't keep adding mgmt sphere tests. I'll approve because it's blocking cat, but we should actively be moving to getting rid of this code